components/service/pocket/src/test/java/mozilla/components/service/pocket/ext/JSONKtTest.kt

```diff
@@ -0,0 +1,63 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.pocket.ext
+
+import org.json.JSONArray
+import org.json.JSONObject
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+class JSONKtTest {
+
```

Comment:
The methods in this test mention `flatMapObj` - but they are testing `mapObjNotNull` :)

Reply:
Oi! IDE refactors! Fixed when I moved the code to the new class.